src/components/formDataTable/FormDataTable.js

```diff
@@ -0,0 +1,103 @@
+import React, { Component } from 'react';
+import BootstrapTable from 'react-bootstrap-table-next';
+import paginationFactory from 'react-bootstrap-table2-paginator';
+import '../datatable/DataTable.scss';
+
+class FormDataTable extends Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      sizePerPage: 10,
+      selected: []
+    }
+    this.handleOnSelect = this.handleOnSelect.bind(this);
+    this.handleOnSelectAll = this.handleOnSelectAll.bind(this);
+  }
+
+  handleOnSelect = (row, isSelect) => {
+    if (isSelect) {
+      this.setState({
+        selected: [...this.state.selected, row.form_id]
+      });
+    } else {
+      this.setState({
+        selected: this.state.selected.filter(x => x !== row.form_id)
+      });
+    }
+  }
+
+  handleOnSelectAll = (isSelect, rows) => {
+    const ids = rows.map(r => r.form_id);
+    if (isSelect) {
+      this.setState({
+        ...this.state,
+        selected: ids
+      });
+    } else {
+      this.setState({
+        ...this.state,
+        selected: []
+      });
+    }
+  }
+
+  render() {
+
+    const sizePerPageRenderer = ({
```

Comment:
We might want to create a wrapper component to add common logic between FormTable and UserTable. Since this is just repeating the code and makes it less maintainable.